pkg/loopback/loopback_test.go

```diff
@@ -0,0 +1,136 @@
+// +build linux,cgo
+
+package loopback
+
+import (
+	"flag"
+	"io/ioutil"
+	"os"
+	"strings"
+	"syscall"
+	"testing"
+
+	"fmt"
+	"github.com/pkg/errors"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+const filePath = "/tmp/"
+
+var rootEnabled bool
+
+func init() {
+	flag.BoolVar(&rootEnabled, "test.root", false, "enable tests that require root")
```

Comment:
We still rely on the uid. There's an assert right after this!  At least in the `pkg/term` case I don't think there was anything destructive. What is destructive here?  And wouldn't skipping the tests with you don't have permissions be safer? When would you have uid == 0 and be the wrong user?

If we absolutely need a switch to turn this on or off (which I'm not convinced we do), we should use an environment variable.